js/utils.js

```diff
@@ -0,0 +1,33 @@
+function getIntRandomNumber(FirstNumber, SecondNumber) {
```

Comment:
названия агрументов следует писать с маленькой буквы
```suggestion
function getIntRandomNumber(firstNumber, secondNumber) {
```